exonum/src/node/basic.rs

```diff
@@ -127,7 +127,7 @@ impl NodeHandler {
         }
         self.state.add_peer(public_key, message.clone());
         info!(
-            "Received Connect message from {}, {}",
+            "Received Connect message from {}. Need to connect: {}",
```

Comment:
Maybe `Need to connect back:` is a little bit more clear?

Reply:
You are right, we can receive `connect`-messages of other nodes in response to `PeersRequest`.